coralproject/talk
# [CORL-1248] Allow users to auto-show media in stream

Allows user to select whether they want to automatically show media or not.

~@wyattjoh, have a look at the schema and let me know if you want things moved around or renamed please!~

## How to test

- Sign in
- Enable `Always show GIFs, Tweets, YouTube, etc.` under My Profile > Preferences.
- Go to your stream
- Post a gif
- See that the gif is pre-unrolled/opened
- Refresh
- See that the gif is still pre-unrolled/opened

- Disable the option under Preferences again and see your gifs are hidden again
